client/app/dedicated/server/bandwidth/order-private/dedicated-server-bandwidth-vrack.html

```diff
@@ -0,0 +1,58 @@
+<div data-wizard
+     data-wizard-on-cancel="cancel"
+     data-wizard-on-finish="order"
+     data-wizard-title=":: 'server_order_bandwidth_vrack_title' | translate"
+     data-wizard-bread-crumb
+     data-wizard-confirm-button-text=":: 'wizard_pay' | translate">
+
+    <div data-wizard-step
+         data-wizard-step-valid="$ctrl.steps[0].isValid()"
+         data-wizard-step-on-load="initFirstStep">
+        <div class="text-center" data-ng-if="$ctrl.steps[0].isLoading()">
+            <oui-spinner></oui-spinner>
+            <span data-translate="server_order_bandwidth_vrack_load" class="d-block"></span>
+        </div>
+        <span data-ng-if="$ctrl.plans.length === 0" data-translate="server_order_bandwidth_empty_plans"></span>
+        <form data-ng-if="!$ctrl.steps[0].isLoading() && $ctrl.plans.length > 0">
+            <p data-translate="select_bandwidth_vrack_type_label"></p>
+            <div data-ng-repeat="plan in $ctrl.plans track by $index">
+                <div class="row">
+                    <div class="col-md-9 pr-0">
+                        <oui-radio data-name="bandwidth-plan" data-model="$ctrl.model.plan" data-value="plan.planCode">
+                            <span data-translate="{{:: 'plan_' + plan.planCode }}"></span>
+                        </oui-radio>
+                    </div>
+                    <div class="col-md-3 pl-0">
+                        <span class="red bold" data-ng-bind-html="plan.prices[2].price.text"></span>
+                        <span class="small" data-translate="plan_price_per_month"></span>
+                    </div>
+                </div>
+            </div>
+            <p class="small" data-translate="server_order_bandwidth_declaration"></p>
+        </form>
+    </div>
+    <div data-wizard-step
```

Comment:
Modal on more than one steps are not allowed, does the mock-up tell you to do this ?

Reply:
The modals are a bit refactored to be used in 2 differents pages.
There's nothing about updating that for now.